scenarios/08_Outlaw_Base.cfg

```diff
@@ -192,7 +191,13 @@
         {MSG_Ruvio _"Hmm... I wonder what might lie in that northern, empty castle..."}
         [objectives]
             side=1
+#ifdef NIGHTMARE
+            #po: no hints for NIGHTMARE difficulty:
+            {OBJECTIVE_NOTES _"Your recruitment of spearmen and archers is limited in this scenario; you will have to find out what exactly the limits are yourself."}
```

Comment:
You could actually just make it really curt and only have the part before the semicolon. Up to you.

Reply:
true, I think I'll do that.

Reply:
done in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).